File: src/main/java/com/github/njuro/jard/ban/BanStatus.java
package com.github.njuro.jard.ban;

/** Enum representing current status of a {@link Ban}. */
public enum BanStatus {

  /** Active ban - IP cannot post. */
  ACTIVE,
  /** Expired ban - IP can post again. */
  EXPIRED,
  /** Ban invalidated by user - IP can post again. */
  UNBANNED,
  /** Ban is just a warning - IP can post, but will be warned. */
  WARNING
}

File: src/main/kotlin/com/github/njuro/jard/rewrite/ban/BanStatus.kt
package com.github.njuro.jard.rewrite.ban

/** Enum representing current status of a [Ban].  */
enum class BanStatus {
    /** Active ban - IP cannot post.  */
    ACTIVE,

    /** Expired ban - IP can post again.  */
    EXPIRED,

    /** Ban invalidated by user - IP can post again.  */
    UNBANNED,

    /** Ban is just a warning - IP can post, but will be warned.  */
    WARNING
}